tests/e2e/gateway/00-install.yaml

```diff
@@ -0,0 +1,157 @@
+---
```

Comment:
I think it would be a good idea if we split this file into two steps:
1. Create the Secrets, ConfigMaps, and other needed stuff and start the app...
2. Create the TempoStack instance

This will help the debuggability of the test. Why? KUTTL has not the best output. If we are checking the creation of the different configurations + the deployment of the Tempo component, it can be difficult to find if what failed is something related to the "needed ingredients" or something the Tempo Operator should have created.

Reply:
sure, i will do.